src/sections/user/user-table-row.tsx

```diff
@@ -13,25 +13,27 @@ import MenuItem, { menuItemClasses } from '@mui/material/MenuItem';
 import { Label } from 'src/components/label';
 import { Iconify } from 'src/components/iconify';
 
-// ----------------------------------------------------------------------
+// ----------------------------------------------------------------
 
 export type UserProps = {
```

Comment:
you are also declaring same UserProps in ```src/sections/location/location-table-row.tsx```
do declare same types or interfaces repeatedly
instead declare one common interface or type in ```src/interface``` folder within relevant file, and export that interface or type, to make importable into any file